### PR TITLE
ZTBoards Updated keymap and updated info.json

### PR DESCRIPTION
**Pull request by phlopsuh**
## Description

Updated information on info.json and hopefully de-cluttered branch, to make merging easier for you guys. 
Also, changed default encoder settings for default keymap. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ztboards/after/info.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
```
Comment (fauxpark) [inline keyboards/ztboards/after/info.json:1]: The ordering seems to be wrong here. You can verify it in Configurator by pressing Ctrl+Shift+I to preview the info.json, and turning on Fast Input.
```json
{
    "keyboard_name": "ZTBoards After",
    "url": "ztb.mx",
    "maintainer": "ZTBoards phlop",
    "width": 16.5,
    "height": 5.75,
    "layouts": {
        "LAYOUT_all": {
            "layout": [
                {"x": 0, "y": 0.5},
                {"x": 1, "y": 0.5},
                {"x": 2, "y": 0.5},
                {"x": 3, "y": 0.5},
                {"x": 4, "y": 0.5},
                {"x": 5, "y": 0.5},
                {"x": 6, "y": 0.5},
                {"x": 7, "y": 0.5},
                {"x": 8, "y": 0.5},
                {"x": 9, "y": 0.5},
                {"x": 10, "y": 0.5},
                {"x": 11, "y": 0.5},
                {"x": 12, "y": 0.5},
                {"x": 13, "y": 0.5},
                {"x": 14, "y": 0.5},
                {"x": 15.5, "y": 0},

                {"x": 0, "y": 1.5, "w": 1.5},
                {"x": 1.5, "y": 1.5},
                {"x": 2.5, "y": 1.5},
                {"x": 3.5, "y": 1.5},
                {"x": 4.5, "y": 1.5},
                {"x": 5.5, "y": 1.5},
                {"x": 6.5, "y": 1.5},
                {"x": 7.5, "y": 1.5},
                {"x": 8.5, "y": 1.5},
                {"x": 9.5, "y": 1.5},
                {"x": 10.5, "y": 1.5},
                {"x": 11.5, "y": 1.5},
                {"x": 12.5, "y": 1.5},
                {"x": 13.5, "y": 1.5, "w": 1.5},

                {"x": 15.5, "y": 1.5},

                {"x": 0, "y": 2.5, "w": 1.75},
                {"x": 1.75, "y": 2.5},
                {"x": 2.75, "y": 2.5},
                {"x": 3.75, "y": 2.5},
                {"x": 4.75, "y": 2.5},
                {"x": 5.75, "y": 2.5},
                {"x": 6.75, "y": 2.5},
                {"x": 7.75, "y": 2.5},
                {"x": 8.75, "y": 2.5},
                {"x": 9.75, "y": 2.5},
                {"x": 10.75, "y": 2.5},
                {"x": 11.75, "y": 2.5},
                {"x": 12.75, "y": 2.5, "w": 2.25},

                {"x": 15.5, "y": 2.5},

                {"x": 0, "y": 3.5, "w": 2.25},
                {"x": 2.25, "y": 3.5},
                {"x": 3.25, "y": 3.5},
                {"x": 4.25, "y": 3.5},
                {"x": 5.25, "y": 3.5},
                {"x": 6.25, "y": 3.5},
                {"x": 7.25, "y": 3.5},
                {"x": 8.25, "y": 3.5},
                {"x": 9.25, "y": 3.5},
                {"x": 10.25, "y": 3.5},
                {"x": 11.25, "y": 3.5},
                {"x": 12.25, "y": 3.5, "w": 1.75},

                {"x": 14.25, "y": 3.75},

                {"x": 15.5, "y": 3.5},

                {"x": 0, "y": 4.5, "w": 1.5},

                {"x": 2.25, "y": 4.5, "w": 1.5},
                {"x": 3.75, "y": 4.5, "w": 6.25},
                {"x": 10, "y": 4.5, "w": 1.5},
                {"x": 11.5, "y": 4.5, "w": 1.5},

                {"x": 13.25, "y": 4.75},
                {"x": 14.25, "y": 4.75},
                {"x": 15.25, "y": 4.75}
            ]
        }
    }
}
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/ztboards/after/info.json (hunk 1 above)
Comment (phlopsuh) [reply keyboards/ztboards/after/info.json:1]: Sorry, could you clarify what you mean by "the ordering seems to be wrong"?

Comment (fauxpark) [reply keyboards/ztboards/after/info.json:1]: When converting KLE to the QMK info.json format, the ordering for keys not on the 1u vertical grid will be incorrect, as they are considered by KLE to be on their own row. In this case the rotary encoder key and the up arrow are one row off:
image
If you turn on Fast Input in the Configurator and spam a key, you can check that the ordering is correct vs. the top half of the LAYOUT macro.

Comment (phlopsuh) [reply keyboards/ztboards/after/info.json:1]: Hello, please check now, it should be in the correct order.

**Review by fauxpark — APPROVED** [review]

File: keyboards/ztboards/after/info.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
     "keyboard_name": "ZTBoards After", 
-    "url": "ztboards.myshopify.com", 
-    "maintainer": "phlopsuh", 
+    "url": "ztb.mx", 
+    "maintainer": "ZTBoards phlop",
```
Comment (noroadsleft) [inline keyboards/ztboards/after/info.json:4]: The value here is intended to be the maintainer's GitHub name.

```suggestion
    "maintainer": "phlopsuh", 
```

File: keyboards/ztboards/noon/info.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
     "keyboard_name": "ZTBoards Noon", 
-    "url": "ztboards.myshopify.com", 
-    "maintainer": "phlopsuh", 
+    "url": "ztb.mx", 
+    "maintainer": "ZTBoards phlop",
```
Comment (noroadsleft) [inline keyboards/ztboards/noon/info.json:4]:
```suggestion
    "maintainer": "phlopsuh", 
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: keyboards/ztboards/noon/info.json (hunk 3 above)
Comment (phlopsuh) [reply keyboards/ztboards/noon/info.json]: Ah okay! should be fixed now.

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**